### DISCUSSION TITLE
The pupil style "Rounded Pointing Out" is missing

### DISCUSSION THREAD

**Discussion by vicvoh** [dagronf/QRCode#61] [Q&A] [answered]
Hi Darren!

Is it on purpose or is it just an error that `QRCode.PupilShape.RoundedPointingOut()` is missing after I updated from ver. 23.3.0 to 25.0.0?

**Comment by dagronf** [comment, marked as answer]
Hi @vicvoh 

I removed `roundedPointingOut` because the same behaviour can be produced by setting `RoundedPointingIn(flip: .both)`. Given that its caused a few people to trip I'll add it back as a convenience.

Grab version 25.0.1 (I just pushed it up) and it should resolve this issue.
